bin/ruby-build

```diff
@@ -701,7 +701,7 @@ fix_rbx_irb() {
 }
 
 require_java7() {
-  local version="$(java -version 2>&1 | grep 'java version' | head -1)"
+  local version="$(java -version 2>&1 | grep '\(java\|openjdk\) version' | head -1)"
```

Comment:
Because of backslashes, this works for me on BSD grep, and since it passes CI (on Linux) I presume with GNU grep as well. It doesn't need changing right now.